homeworks/vazgen_harutyunyan/Homework_2/Problem59.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+public class Problem59 {
+    public static void main(String[] args) {
+        int number;
+        int firstDigit;
+        int secondDigit;
+        int lastDigit;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your number");
+        number = scanner.nextInt();
+        lastDigit = number % 10;
+        number = number / 10;
+        secondDigit = number % 10;
+        number = number / 10;
+        firstDigit = number;
+        if(firstDigit > secondDigit && firstDigit > lastDigit) {
```

Comment:
the problem is solved, but it can be simplified if you use one comparison in condition